styles.css

```diff
@@ -13,19 +13,370 @@
 ////////////////////////*/
 
 /* ------------------------------- 1. Global Styles -------------------------------- */
+@import url('https://fonts.googleapis.com/css2?family=Merriweather&family=Roboto:wght@300;400;700&display=swap');
+
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    /*border: 1px solid red;*/
+}
+
+
+/*Kleuren*/
+
+:root {
+    --color-purple: #5722CD;
+    --color-blue: #4845E4; /*header background*/
+    --color-pink: #ED4385;
+    --color-black-standard-text: #0C0C0C; /*standaard tekst*/
+    --color-light-purple-big-floating-words: #F2F0FF;
+    --color-light-grey-background-projects: #F3F6F9;
+    --color-grey-service-tile: #7A7A7A;
+}
+
 
 /* ------------------------------- 2. Typography ----------------------------------- */
 
+body {
+    font-family: 'Roboto', Arial, sans-serif;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: 'Merriweather', Arial, serif;
+}
+
+h1, h2 {
+    font-size: 2rem;
+    padding: 2rem 0;
+}
+
+h3 {
+    font-size: 1.5rem;
+    padding: 1.5rem 0;
+}
+
+h4 {
+    font-size: 1.3rem;
```

Comment:
Controleer altijd of je rem waardes overeenkomen met hele pixels. Dit komt namelijk neer op een kommagetal. Dat is niet lekker voor een browser om te renderen (laat staan, schalen)